stable/yugaware/templates/configs.yaml

```diff
@@ -582,6 +582,8 @@ data:
 
       {{- end }}
       {{- end }}
+ 
+      {{- if not .Values.tls.enabled }}
```

Comment:
I think we should also use this block if `useNginxProxy` is enabled. 

Basically, if someone is using nginx proxy, then the YBA will be listening on 9000 (http).

Reply:
Please check @bhavin192 , I have addressed your comment.